src/wp-admin/includes/file.php

```diff
@@ -1946,6 +1946,68 @@ function copy_dir( $from, $to, $skip_list = array() ) {
 	return true;
 }
 
+/**
+ * Moves a directory from one location to another via the rename() PHP function.
+ * If the renaming failed, falls back to copy_dir().
+ *
+ * Assumes that WP_Filesystem() has already been called and setup.
+ *
+ * @since 6.2.0
+ *
+ * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
+ *
+ * @param string $from        Source directory.
+ * @param string $to          Destination directory.
+ * @return true|WP_Error True on success, WP_Error on failure.
+ */
+function move_dir( $from, $to ) {
+	global $wp_filesystem;
+
+	$result = false;
+
+	/**
+	 * Fires before move_dir().
+	 *
+	 * @since 6.2.0
+	 */
+	do_action( 'pre_move_dir' );
+
+	if ( 'direct' === $wp_filesystem->method ) {
+		$wp_filesystem->rmdir( $to );
+
+		$result = @rename( $from, $to );
+	}
+
+	// Non-direct filesystems use some version of rename without a fallback.
+	if ( 'direct' !== $wp_filesystem->method ) {
```

Comment:
Repeat conditional instead of `else`?

Reply:
Could certainly use `else` here. At the time it just looked cleaner to read and understand. At least it did when I added it.